syntax = "proto3";

option java_package = "ex.grpc";

package test_message;

service TestMessageService {
    rpc sendRequest (TestMessageRequest) returns (TestMessageResponse) {}
}

message TestMessageRequest {
    int32 a = 1;
    int32 b = 2;
}

message TestMessageResponse {
    int32 result = 1;
}